build.sbt

```diff
@@ -616,18 +667,17 @@ lazy val test = project
     fork in IntegrationTest := true,
     javaOptions in IntegrationTest += "-Xmx1G",
     testFrameworks += new TestFramework("scala.tools.partest.sbt.Framework"),
-    testOptions in IntegrationTest += Tests.Setup( () => root.base.getAbsolutePath + "/pull-binary-libs.sh" ! ),
```

Comment:
OK, agree.could we maybe add an assertion somewhere in the sbt startup, require that `test/files/lib/annotations.jar` exists?